deploy/chart/templates/job.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: sap-btp-operator-migration
   namespace: {{.Release.Namespace}}
 spec:
-  backoffLimit: 6
+  backoffLimit: 30
```

Comment:
sometimes reconciler takes time installing btp-operator, increasing backoff limit helps with allowing this to wait for the operator to be ready